hkmc2/shared/src/test/mlscript/codegen/Getters.mls

```diff
@@ -0,0 +1,256 @@
+:js
+
+:import PredefJS.mls
```

Comment:
Oh, this is outdated. I will remove this file later. There should be no need to import it.